code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -165,14 +167,15 @@
 	name = "Syndicate Officer"
 	alert_on_death = TRUE
 	alert_on_timeout = TRUE
-
+	melee_block_chance = 60
 
 /mob/living/simple_animal/hostile/syndicate/melee/autogib/depot/armory
 	name = "Syndicate Armory Officer"
 	icon_state = "syndicatemeleespace"
 	icon_living = "syndicatemeleespace"
 	maxHealth = 250
 	health = 250
+	melee_block_chance = 80

```

Comment:
This man is ROBUST

Reply:
It used to be they were all effectively immune to melee damage. Now, I've made it scale - the further into the depot you go, the harder the mobs get - in resistance to melee, and other ways.